client/src/components/pages/MainChannel.tsx

```diff
@@ -126,6 +134,11 @@ const ChannelsFragment: FC<ChannelProps> = ({channel, searchArgs}) => {
             channelId: item.node.id,
           });
         }}
+        onLongPress={(): void => {
+          showLeaveModal({
+            channelId: item.node.id,
```

Comment:
#### Should Fix
Is there a reason why `showLeaveModal` takes an object as a parameter? I think it would be simpler if the function takes a string as its parameter (i.e. `showLeaveModal(item.node.id)`).

Reply:
Thank you!
I will modify it!